packages/core/test/request-builder/update-request-builder.spec.ts

```diff
@@ -207,7 +200,7 @@ describe('UpdateRequestBuilder', () => {
     expect(actual).toEqual(entity.setOrInitializeRemoteState());
   });
 
-  it('update request should contain verson identifier when set on request', async () => {
+  it('update request should contain version identifier when set on request', async () => {
```

Comment:
when you already change it perhaps you can also rephrase it in a "it(should...)" sentence.